tests/app/main/test_login.py

```diff
@@ -1,24 +1,23 @@
-from nose.tools import assert_equal, assert_true, \
-    assert_is_not_none
+from app.main import helpers
+from nose.tools import assert_equal, assert_true, assert_is_not_none
 from ..helpers import BaseApplicationTest
 from mock import Mock
 from app import data_api_client
 
 
+email_empty_error = "Email can not be empty"
+email_invalid_error = "Please enter a valid email address"
+email_sent_message = "If that Digital Marketplace supplier account exists, " \
+                     "you will be sent an email containing a link to reset " \
+                     "your password."
+password_empty_error = "Please enter your password"
+password_invalid_error = "Passwords must be between 10 and 50 characters"
+password_mismatch_error = "The passwords you entered do not match"
+new_password_empty_error = "Please enter a new password"
+new_password_confirm_empty_error = "Please confirm your new password"
```

Comment:
OK will do - I was just modifying the constants already there, which were lower to start with.